changelog.d/2743.doc.rst

```diff
@@ -0,0 +1 @@
+Set .editorconfig `max_line_length` property for Python files.
```

Comment:
Use RST syntax, not markdown:
```suggestion
Set :file:`.editorconfig` ``max_line_length`` property for Python files.
```

Reply:
Thank you, my bad. My brain was indeed in Markdown mode.